### PR TITLE
WIP: Change default colors for fNIRS

### PR DESCRIPTION
**Pull request by rob-luke**
#### What does this implement/fix?
Change the color of hbo to red. It is traditional in fnirs literature for hbo to be red and hbr to be blue.

### CODE REVIEW

File: mne/defaults.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     color=dict(mag='darkblue', grad='b', eeg='k', eog='k', ecg='m', emg='k',
                ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
-               gof='k', bio='k', ecog='k', hbo='darkblue', hbr='b',
+               gof='k', bio='k', ecog='k', hbo='r', hbr='b',
```
Comment (agramfort) [inline mne/defaults.py]: is there a chance that this is confounded by red which means bad epoch in epochs.plot or evoked.plot when you show bads?

Comment (rob-luke) [reply mne/defaults.py]: Sounds like that's going to be a problem! (I haven't got to evoked.plot yet).

I was just trying to bring MNE inline with convention for fNIRS. My goal over the next few weeks is to get to a plot that looks like this...

image

The plan was to figure out how to make default colors match convention, then figure out how to plot two different types on the same figure. But sounds like changing the raw traces to red might be a bad idea. Would a different red be possible? If not, lets leave as is and drop this PR.

Comment (larsoner) [reply mne/defaults.py]: > Would a different red be possible? If not, lets leave as is and drop this PR.

Yes I think some other shade of red could work. Maybe a dark red or orangeish or purpelish red? Maybe `#AA3377`? I often look at [this PDF](https://personal.sron.nl/~pault/data/colourschemes.pdf) for color ideas

Comment (larsoner) [reply mne/defaults.py]: > I was just trying to bring MNE inline with convention for fNIRS.

... as I think we should try to follow accepted convention as much as possible. So if we can get something "red enough" to not surprise and/or annoy fNIRS folks and also something sufficiently "not red" that our bad channel marking (which is effectively `#FF0000` IIRC) still stands out we should be good.

**Comment by larsoner** [comment]
Thanks @rob-luke. Once we have a tutorial with lots of plotting functions we can evaluate and tweak further if necessary

**Comment by rob-luke** [comment]
So I finally got around to recording some data for a tutorial. Its currently living [here](https://github.com/rob-luke/mne-fnirs-demo/tree/master/tapping-P1), and the beginnings of a tutorial are [here](https://github.com/rob-luke/mne-fnirs-demo/blob/master/Tapping.ipynb).

@larsoner : where is best for me to put this recording so that it can be used in an MNE tutorial?

The data is from optodes placed over the motor cortex. There are three conditions, tapping the left thumb to fingers, tapping the right thumb to fingers, and a control where nothing happens. The tapping lasts 5 seconds. 30 trials of each condition are done. I included short channels on all sources, so that when we implement functionality for that in MNE, we can still use this dataset.

**Comment by larsoner** [comment]
> @larsoner : where is best for me to put this recording so that it can be used in an MNE tutorial?

We usually upload to osf.io under [MNE's group](https://osf.io/rxvq7/). You can structure your data like:
```
MNE-fNIRS-motor-data
MNE-fNIRS-motor-data/my_filename_whatever.nirx
MNE-fNIRS-motor-data/...
```
By convention we usually name the folder `MNE-*-data`, feel free to choose a better `*`. You can structure with subdirectories or not if you want (`MNE-sample-data` has `subjects` and `MEG` subfolders, `MNE-multimodal-data` has a single file `multimodal_raw.fif`.) Then `tar xzfv MNE-fNIRS-motor-data.tgz MNE-fNIRS-motor-data`, and upload somewhere I can get to it to upload it, then I can add it to MNE's osf.io share. osf.io has versions, too, in case we need to update it (see how we have `version` in the other osf.io URLs in `datasets/utils.py`.

Once that's done, you'll need to add things a few places. If you `git grep multimodal`, it should show you all the places you need to update (python_reference.rst, datasets_index.rst, making a new folder, adding hash, updating `.circleci/config.yml`, `_download_all_example_data`, etc.). It will look like a lot but really it's probably 10-15 minutes of copy-paste and changing names/adapting code.

**Comment by rob-luke** [comment]
@larsoner I have generated the file and uploaded it [here](https://osf.io/uq8sd/).

**Comment by larsoner** [comment]
I have you write permissions to the MNE repo on OSF

**Comment by larsoner** [comment]
(just now I have them to you I mean)

**Comment by rob-luke** [comment]
Done. Thanks

**Pull request merged**